tests/test_run_inputs.py

```diff
@@ -0,0 +1,129 @@
+from typing import Optional
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.remote.webdriver import WebDriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.common.exceptions import (
+    ElementNotVisibleException,
+    ElementNotSelectableException,
+    NoSuchElementException,
+)
+from selenium.webdriver import ActionChains
+
+web = "https://machinist.flapflap.io/"
+run_inputs_path = "#__next > main > div.MenuBar__Root-sc-1th5wld-0.jDqDxJ.App__FloatingMenuBar-gv0e21-2.eZsxUV > ul > li:nth-child(4)"
+
+
+def test_run_input_header(driver: WebDriver):
+
+    driver.get(web)
+    driver.find_element(By.CSS_SELECTOR, run_inputs_path).click()
+
+    wait = WebDriverWait(
```

Comment:
We are creating the same `WebDriverWait` here and on line 110. We should unify these. I would recommend a factory function that creates `WebDriverWait` instances rather than making this `wait` variable global.

The reason to use a factory function instead of just making a shared `WebDriverWait` is that we may want to apply some additional configurations to the `WebDriverWait` in certain tests. If we make it a shared instance, then those additional configurations will apply to all tests, which is not desirable.

So making a factory function with a signature like:

```python
def wait(driver: WebDriver, *ignored_exceptions: WebDriverException) -> WebDriverWait: ...
```